doc/cug.tex

```diff
@@ -7468,6 +7468,31 @@ \subsection{No Implicit Creation of Tasks by Offset Triggers}
 because it is error prone: if a trigger offset is wrong it should cause a
 triggering failure rather than create task instances at incorrect cycle points.
 
+\section{Known Issues}
+\label{KnownIssues}
+
+\subsection{Current Known Issues}
+\label{CurrentKnownIssues}
+
+The best place to find current known issues is on Github:
+\url{https://github.com/cylc/cylc/issues}.
+
+\subsection{Other Known Issues}
```

Comment:
Could we name this "Notable Known Issues" - that way we can use the section for documenting things like the Family triggering memory overhead and the GraphViz GTK memory leak.